.jules/palette.md

```
@@ -0,0 +1,6 @@
+## 2026-04-14 - TUI Help Footer Alignment and Discoverability
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)